indico/modules/events/registration/client/js/form/selectors.js

```diff
@@ -55,3 +57,22 @@ export const getNestedSections = createSelector(
       items: sectionFields.get(section.id),
     }))
 );
+
+export const getFieldLabelLookup = createSelector(
+  getItems,
+  items => {
+    const lookup = new Map();
+    for (const formField of Object.values(items)) {
+      lookup.set(formField.htmlName, {
+        label: formField.title,
+        id: `input-${formField.id}`,
+      });
+    }
+    // Include the "Captcha" field which isn't part of reg form registry
```

Comment:
I think the "Specify whether you consent to being included in the event's list of participants" field (when the participant list is configured to use Consent in the privacy settings) needs to be handled in a similar way here.